apps/projects/app/components/Content/IssueDetail/EventsCard.js

```diff
@@ -53,13 +53,13 @@ IssueEvent.propTypes = {
 
 const applicationLink = (user, onReviewApplication, issue, index) => (
   <React.Fragment>
-    {user} submitted <Link onClick={() => onReviewApplication(issue, index)}>an application for review</Link>
+    {user} submitted <Link onClick={() => onReviewApplication(issue, index, true)}>an application for review</Link>
```

Comment:
I find it really hard to understand what a function will do with a raw boolean like this, and would prefer that we passed it with a name. We can do that like this:

```js
onReviewApplication({ issue, index, readOnly: true })
```

Reply:
Changed in all places for both panels. Actually easier than it seemed.